stylix/maintainers.nix

```
@@ -16,4 +16,11 @@
       { fingerprint = "36BC 916D DD4E B1EE EE82  4BBF DC95 900F 6DA7 9992"; }
     ];
   };
+
+  skoove = {
+    email = "zie@sturges.com.au";
+    name = "Zie Sturges";
+    github = "skoove";
+    githubId = 	53106860;
```

> ```diff
> -    githubId =      53106860;
> +    githubId = 53106860;
> ```

The formatting CI is failing:

```suggestion
    githubId = 53106860;
```